server/game/core/ongoingEffect/OngoingEffect.ts

```diff
@@ -148,4 +174,4 @@ class OngoingEffect {
     }
 }
 
-module.exports = OngoingEffect;
+export default OngoingEffect;
```

Comment:
We haven't really been using export default a lot anyway, could we just change to using `export class OngoingEffect` and then update relevant imports to be `{ OngoingEffect }`? This would be the only TS file in the repo that has the export at the end of the file like this

Reply:
Can do, I had to do that for OngoingEffectSource anyway.